src/i18n/es/ui.ts

```diff
@@ -1,113 +1,122 @@
 import { UIDictionary } from '../translation-checkers';
 
 export default UIDictionary({
-	'a11y.skipLink': 'Saltearse al contenido',
+	'a11y.skipLink': 'Ir al contenido',
 	'a11y.sectionLink': 'Sección titulada',
-	'navbar.a11yTitle': 'Superior',
-	// Aside component default labels
-	'aside.note': 'Nota',
-	'aside.tip': 'Sugerencia',
-	'aside.caution': 'Precaución',
-	'aside.danger': 'Peligro',
-	// Site settings
+	'navbar.a11yTitle': 'Inicio',
+	// Configuración del sitio
 	'site.title': 'Documentación de Astro',
-	'site.description':
-		'Construye páginas web más rápidas con menos JavaScript en el lado del cliente.',
+	'site.description': 'Construye sitios web más rápidos con menos JavaScript en el lado del cliente.',
 	'site.og.imageSrc': '/default-og-image.png?v=1',
 	'site.og.imageAlt':
-		'Logo de Astro en el espacio estrellado, con un planeta púrpura parecido a Saturno flotando en el fondo a la derecha.',
-	// Left Sidebar
+		'Logo de Astro en un espacio estrellado, con un planeta púrpura similar a Saturno flotando en el primer plano derecho',
+	// Barra lateral izquierda
 	'leftSidebar.a11yTitle': 'Primario',
-	'leftSidebar.learnTab': 'Aprende',
-	'leftSidebar.referenceTab': 'Referencias',
+	'leftSidebar.learnTab': 'Aprender',
+	'leftSidebar.referenceTab': 'Referencia',
 	'leftSidebar.viewInEnglish': 'Ver en inglés',
 	'leftSidebar.sponsoredBy': 'Patrocinado por',
-	// Right Sidebar
+	// Barra lateral derecha
 	'rightSidebar.a11yTitle': 'Secundario',
 	'rightSidebar.onThisPage': 'En esta página',
-	'rightSidebar.overview': 'Sinopsis',
+	'rightSidebar.overview': 'Visión general',
```

Comment:
Creo que este cambio no es necesario, personalmente lo dejaria como estaba anteriormente
```suggestion
	'rightSidebar.overview': 'Sinopsis',
```